recipes/sgp4/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "sgp4" %}
+{% set version = "1.4" %}
+{% set gh_org = "brandon-rhodes" %}
+{% set gh_repo = "python-sgp4" %}
+{% set sha256 = "a7e03a714de969e2ad980830957793d677681b40047d7d052a50fe9e96e6ab7c" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/{{ gh_org }}/{{ gh_repo }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - manifest.patch
+
+build:
+  skip: true  # [win64]
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - sgp4
+    - sgp4.tests
+  commands:
+    - python -m unittest -v sgp4.tests
+
+about:
+  home: https://github.com/brandon-rhodes/python-sgp4
+  license: MIT
+#  license_file: LICENSE
```

Comment:
@scopatz Maybe I am stupid but for the life of me I cannot figure out the way to just add the file manually - like I did in 40f2ae7. It did not get recognized by the build system. I have tried to add it to the package by patching but that adds the file to sgp4 package directory and this is also not recognized. Can you help me with this?